ipatests/test_integration/test_testconfig.py

```diff
@@ -60,6 +61,8 @@
     "IPv6SETUP": "",
     "IPADEBUG": "",
     "DOMAINLVL": str(MAX_DOMAIN_LEVEL),
+    "SYSTEMD_PAGER": "cat",  # allows journalctl log collection
```

Comment:
You added this only into test, so test will fail because nothing was added to config code.